pkgs/by-name/co/cosmopolitan/package.nix

```
@@ -11,13 +12,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Would probably be better since we don't risk messing up cosmocc with our own cc.